pkg/deb/build-deb.sh

```diff
@@ -77,13 +77,8 @@ ls $CPP_DIR/lib/libpulsar*
 cp -ar $CPP_DIR/include/pulsar $DEVEL_DEST_DIR/usr/include/
 cp $CPP_DIR/lib/libpulsar.a $DEVEL_DEST_DIR/usr/lib
 cp $CPP_DIR/lib/libpulsarwithdeps.a $DEVEL_DEST_DIR/usr/lib
-cp $CPP_DIR/lib/libpulsar.so.$POM_VERSION $DEST_DIR/usr/lib
-cp $CPP_DIR/lib/libpulsarnossl.so.$POM_VERSION $DEST_DIR/usr/lib
-
-pushd $DEST_DIR/usr/lib
-ln -s libpulsar.so.$POM_VERSION libpulsar.so
-ln -s libpulsarnossl.so.$POM_VERSION libpulsarnossl.so
-popd
+cp $CPP_DIR/lib/libpulsar.so $DEST_DIR/usr/lib
+cp $CPP_DIR/lib/libpulsarnossl.so $DEST_DIR/usr/lib
```

Comment:
Do we need a version suffix?

Reply:
We don't really need them. It would be anyway unsupported to have different versions of the same libpulsar in the system.